extra-network/zerotier-one/autobuild/build

```
@@ -1,4 +1,9 @@
+abinfo "Substitute zerotier-one.service path..."
 sed -i 's/sbin/bin/' make-linux.mk debian/zerotier-one.service
+
+abinfo "Make zerotier-one..."
 make
+
+abinfo "Install zerotier-one..."
 make DESTDIR="${PKGDIR}" install
 install -Dm644 debian/zerotier-one.service "${PKGDIR}"/usr/lib/systemd/system/zerotier-one.service
```

4. `install`, `cp`, `mv` and so on commands have verbose mode, so add verbosity if possible. And don't forget to use absolute path:
```suggestion
install -Dvm644 "${SRCDIR}"/debian/zerotier-one.service \
    "${PKGDIR}"/usr/lib/systemd/system/zerotier-one.service
```

For more infomation, you could read our [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/), it also has [Simplified Chinese version](https://wiki.aosc.io/developer/packaging/package-styling-manual/).

Enjoy packaging! :smile: